newIDE/app/src/Profile/Subscription/PlanCard.js

```diff
@@ -25,6 +25,48 @@ type Props = {|
   background: 'medium' | 'dark',
 |};
 
+const getPlanPrice = ({
+  plan,
+  hidePrice,
+}: {
+  plan: PlanDetails,
+  hidePrice?: boolean,
+}): React.Node => {
+  if (hidePrice || plan.monthlyPriceInEuros === null) return null;
+  if (plan.monthlyPriceInEuros === 0)
+    return (
+      <Text noMargin color="secondary">
+        <Trans>Free</Trans>
+      </Text>
+    );
+  const prices = [];
+  prices.push(
+    plan.isPerUser ? (
+      <Text noMargin color="secondary">
+        <Trans key="month">{plan.monthlyPriceInEuros}€/seat/month</Trans>
```

Comment:
I believe your `key` must be on the `Text` rather than the `Trans` (same for all others)

Reply:
Good spot!